dashboard/management/commands/createkeys.py

```diff
@@ -24,41 +33,51 @@ def add_arguments(self, parser: CommandParser):
                  f'timestamp of "{self.timestampFormat.replace("%", "%%")}"',
             dest=self.baseNameOption)
 
+    def _writeKeyFile(self, keyFileType: KeyFileType, keyContent: str):
+        keyFileName: str = f'{self.keyFileBasePathName}' \
+                           f'{keyFileType.fileSuffix}'
+        self.stdout.write(
+            f'Writing {keyFileType.description} to file '
+            f'"{keyFileName}"...')
+        with open(keyFileName, 'w') as f:
+            f.writelines((keyContent, '\n'))
+        self.stdout.write('File successfully written.')
+
+    @property
+    def keyFileBasePathName(self):
+        return self._keyFileBasePathName
+
+    @keyFileBasePathName.setter
+    def keyFileBasePathName(self, value: str):
+        self._keyFileBasePathName = value
```

Comment:
WRT my commit comment that my change to simplify the code made it longer, it could have been a few lines shorter if I'd left these property methods out.  I think it's better with them in, though.